### PR TITLE
ref(ui): add sidebar to settings

### PR DESCRIPTION
**Pull request by ckj**
This adds the global sidebar to settings and updates the heading styles to match the rest of the app:

screen shot 2019-01-08 at 3 33 19 pm

Known todos:

- [x] Have settings icon active in sidebar when viewing User settings
- [x] Make footer sticky

### CODE REVIEW

File: src/sentry/static/sentry/app/components/sidebar/index.jsx [hunk 1]
```diff
@@ -300,6 +300,16 @@ class Sidebar extends React.Component {
                   to={`/organizations/${organization.slug}/stats/`}
                 />
               </SidebarSection>
+
+              <SidebarSection>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/sidebar/index.jsx]: Should this be feature flagged with `sentry10`? FYI there's one in the sidebar with the `sentry10` flag already.

File: src/sentry/static/sentry/app/views/settings/components/settingsLayout.jsx [hunk 2]
```diff
@@ -43,40 +39,38 @@ class SettingsLayout extends React.Component {
...
 const SidebarWrapper = styled(Box)`
   flex-shrink: 0;
   width: ${p => p.theme.settings.sidebarWidth};
-  margin-top: 7px;
+  background: #fff;
+  border-right: 1px solid ${p => p.theme.borderLight};
+  padding: 30px;
+`;
+
+const SettingsColumn = styled('div')`
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/components/settingsLayout.jsx]: This should be flex with direction column and `MaxWidthContainer` should have `flex: 1`, that'll make footer stick to bottom

Comment (ckj) [reply src/sentry/static/sentry/app/views/settings/components/settingsLayout.jsx]: Ty!

File: src/sentry/static/sentry/app/components/sidebar/index.jsx (hunk 1 above)
Comment (ckj) [reply src/sentry/static/sentry/app/components/sidebar/index.jsx]: I'm leaning towards no flag. there's not really a new feature here, just a slight usablity boost from surfacing the settings icon and making the layout more consistent with the rest of the app

Comment (billyvg) [reply src/sentry/static/sentry/app/components/sidebar/index.jsx]: Let's remove the SidebarItem with the sentry10 flag then

Comment (ckj) [reply src/sentry/static/sentry/app/components/sidebar/index.jsx]: Ah, I didn't realize you meant there was already _Settings_ icon with the flag. TIL. I'll remove it.

File: src/sentry/static/sentry/app/views/settings/components/settingsHeader.jsx [hunk 3]
```diff
@@ -3,25 +3,10 @@ import styled from 'react-emotion';
...
-    bottom: 5px;
-
-    background-image: linear-gradient(
-      to bottom,
-      #fcfcfc 0%,
-      #fcfcfc 80%,
-      rgba(255, 255, 255, 0.2) 100%
-    );
-  }
+  padding: 20px 30px;
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/views/settings/components/settingsHeader.jsx]: `space(3) space(4)`

File: src/sentry/static/sentry/app/views/settings/components/settingsLayout.jsx [hunk 4]
```diff
@@ -43,40 +39,41 @@ class SettingsLayout extends React.Component {
...
+  flex: 1;
 `;
 
 const SidebarWrapper = styled(Box)`
   flex-shrink: 0;
   width: ${p => p.theme.settings.sidebarWidth};
-  margin-top: 7px;
+  background: #fff;
+  border-right: 1px solid ${p => p.theme.borderLight};
+  padding: 30px;
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/views/settings/components/settingsLayout.jsx]: `space(4)`

File: src/sentry/static/sentry/app/views/settings/components/settingsLayout.jsx [hunk 5]
```diff
@@ -43,40 +39,41 @@ class SettingsLayout extends React.Component {
...
+`;
+
+const SettingsColumn = styled('div')`
+  display: flex;
+  flex-direction: column;
+  flex: 1; /* so this stretches vertically so that footer is fixed at bottom */
+  footer {
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/views/settings/components/settingsLayout.jsx]: is it ideal to link this to a generic `footer` element?

File: src/sentry/static/sentry/app/views/settings/components/settingsLayout.jsx [hunk 6]
```diff
@@ -85,15 +82,8 @@ const SidebarWrapper = styled(Box)`
  */
 const Content = styled(Box)`
   flex: 1;
+  padding: 30px;
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/views/settings/components/settingsLayout.jsx]: `space(4)`

File: src/sentry/static/sentry/app/views/settings/components/settingsWrapper.jsx [hunk 7]
```diff
@@ -13,11 +13,6 @@ const StyledSettingsWrapper = styled.div`
   color: ${p => p.theme.gray5};
   margin: 0 auto;
   line-height: 1;
-
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/views/settings/components/settingsWrapper.jsx]: 👏

File: src/sentry/static/sentry/app/views/settings/components/settingsPageHeader.jsx [hunk 8]
```diff
@@ -41,14 +46,20 @@ const Title = styled(Flex, {shouldForwardProp: p => p !== 'styled'})`
     `
     font-size: 20px;
     font-weight: bold;`};
-  margin: ${space(3)} 0;
+  margin: ${p => (p.tabs ? `${space(4)} 0 ${space(2)}` : `${space(4)} 0`)};
   flex: 1;
 `;
 
+const Action = styled('div')`
+  ${p =>
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/views/settings/components/settingsPageHeader.jsx]: if you use a ternary here prettier won't chew this up

**Review by Chrissy — APPROVED** [review]
pending [this convo](https://github.com/getsentry/sentry/pull/11424/files#r246253977) otherwise the styles look good!

**Review by billyvg — APPROVED** [review]

**Pull request merged**